modules/rostests/kmtests/kmtest_drv/kmtest_fsminifilter.c

```diff
@@ -461,7 +463,9 @@ KmtFilterRegisterCallbacks(
         return STATUS_INSUFFICIENT_RESOURCES;
     }
 
-    /* Copy the array, but using the our own pre/post callbacks */
+    CallbacksCount = Count + 1;
```

Comment:
You want `Count` here, the last one doesn't have valid data.

Reply:
Though that changes what existing code seemed to try to do, I agree wrt this PR.
Done. + moved this assignment + added explicit comment + added FIXME comment.